assets/js/components/Modal.js

```diff
@@ -37,7 +37,7 @@ module.exports = class Modal {
   }
 
   modalToggle(event) {
-    if (!this.$elm.classList.contains('small-device') || this.window.innerWidth < 480) {
+    if (!this.$elm.classList.contains('small-device')) {
```

Comment:
How will removing the additional condition affect the behaviour.

Reply:
It is a mistake, we need to keep the old logic for the share modal to be visible only on small screens but if we remove the class "small-device", we can reuse the logic for the new button and it will be visible on all screen sizes, the condition is reverted, there is no point of removing it